google/cloud/storage/internal/storage_stub_factory_test.cc

```diff
@@ -27,10 +28,15 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace internal {
 namespace {
 
+using ::google::cloud::testing_util::ScopedLog;
 using ::google::cloud::testing_util::StatusIs;
+using ::testing::Contains;
+using ::testing::HasSubstr;
 
 // All the tests have nearly identical structure. They create a stub pointing to
-// an invalid endpoint. Getting a kUnavailable error is the expected result.
+// an invalid endpoint. The stubs have full decorators. We expect that they
+// will result in a call to a service.  Getting a kUnavailable error is the
```

Comment:
Maybe combine these two sentences. "We expect that they will make a call to the service which results in a kUnavailable error."

Or just get rid of the "We expect that they will result in a call to a service" sentence altogether.

Reply:
I changed the comments a bit.  I am going to make a couple of big changes to these tests, so getting the language "just so" is not that important.

Reply:
Meaning: thanks for the suggestions, they are good, but I am going to keep changing the comments anyway.